src/components/DashboardResultIndicatorsSection.vue

```diff
@@ -126,6 +126,12 @@ const getResultIndicatorPeriods = () => {
   sixMonthsBack.setMonth(sixMonthsBack.getMonth() - 6);
 
   return {
+    all: {
```

Comment:
Burde årlig også flyttes før 6-månederen nå? :sweat_smile: Sånn at de er sorterte?

Reply:
Tjoo, tenkte på det, og du tenker da fra lengste til minste intervall? Tenkte bare "fra første" ikke *nødvendigvis* er "større", og den vi hadde som default nå (og antatt mest nyttig til nå), havner nederst. Men siden du reagerer er det kanskje nødvendig å endre.

Reply:
Ja, den er jo ikke størst de første seks månedene, men føles størst allikevel? :sweat_smile: Vet ikke, men føltes naturlig for meg i hvert fall.